main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"github.com/peterbourgon/ff/v3"
+	"github.com/peterbourgon/ff/v3/fftoml"
+	"github.com/redhatinsights/rhc-worker-playbook/internal/config"
+	"github.com/redhatinsights/rhc-worker-playbook/internal/constants"
+	"github.com/redhatinsights/yggdrasil/worker"
+)
+
+func main() {
+	fs := flag.NewFlagSet(filepath.Base(os.Args[0]), flag.ExitOnError)
```

Comment:
Please, let's not lock ourselves into this `Go`ism of string flags with a single dash. This is a public API we have to keep, let's use `--flag`-style configuration using some library.

Reply:
For user-facing software like `yggctl`, `yggd, and `rhc`, I completely agree with you. I'd use a more fully-featured library. For software like this, that isn't meant to be run frequently by users manually, I'm less concerned with the fluency of the API. Using the standard library's `flag` package means fewer third-party package dependencies. I find that preferable in cases like this where the user experience isn't as high a priority.

Reply:
And for what it's worth, double-dash flags are parsed as equivalent to the single-dash flags.

```console
$ /usr/libexec/rhc-worker-playbook --version
0.2.0
```

Reply:
I refactored `main` to use `urfave/cli` instead of `ff`.